lib/api/multipartDelete.js

```diff
@@ -61,8 +62,12 @@ function multipartDelete(authInfo, request, log, callback) {
         function checkMPUval(destBucket, next) {
             services.metadataValidateMultipart(metadataValParams,
                 (err, mpuBucket, mpuOverviewArray) => {
-                    if (err) {
-                        return next(err, destBucket);
+                    if (err === errors.NoSuchUpload) {
+                        log.trace('did not find valid mpu with uploadId' +
+                        `${uploadId}`, { method: 'multipartDelete' });
+                        // TODO in rel/6.3 and later: return 404 if
+                        // usEastBehavior is enabled in config
+                        return callback(null, destBucket);
```

Comment:
yeah sorry this is wrong, we want to collect corsHeaders first and return `callback(null, corsHeaders)`